integration/update_test.go

```diff
@@ -628,6 +629,46 @@ func TestUpdateMany(t *testing.T) {
 				Message: "Updating the path 'foo' would create a conflict at 'foo'",
 			},
 		},
+		"UnsetString": {
```

Comment:
I think tests for ’$unset’ should be in ’TestUpdateUnsetOperator’, not in ’TestUpdateMany’.

Reply:
> I think tests for `$unset` should be in `TestUpdateUnsetOperator`, not in `TestUpdateMany`.

why? 
we were about to create another task to restructure update tests, let's do it once in that PR?